Patch 1 (#56)
howardwu/awesome-aleo: README.md

### Gaming

> - [Where's Alex](https://github.com/puzzlehq/wheres_alex) simple multiparty hidden information game - a game of hide and seek with Alex Pruden! The most forked game on Aleo built using Puzzle SDK for zkGames.

Reviewer: ```suggestion
- [Where's Alex](https://github.com/puzzlehq/wheres_alex) - A multiparty hidden information game on Aleo
```

> - [Treasure Hunt](https://treasures.puzzle.online/) multiparty hidden information game --built using Puzzle SDK for zkGames and Puzzle's game bot infrastructure

Reviewer: ```suggestion
- [Treasure Hunt](https://treasures.puzzle.online/) - A multiparty hidden treasure hunt game on Aleo
```

> - [Super Leo Ligue](https://aleo-fortune.vercel.app/) Play classic roulette and more onchain -- built using Puzzle SDK for zkGames

Reviewer: ```suggestion
- [Super Leo Ligue](https://aleo-fortune.vercel.app/) - Play classic casino games like roulette and more on Aleo
```

> - [Aleo Fortune](https://superleolig.vercel.app/) challenge a friend to an onchain soccer game with hidden strategies -- built using Puzzle SDK for zkGames

Reviewer: ```suggestion
- [Aleo Fortune](https://superleolig.vercel.app/) - Challenge a friend to an onchain soccer game with hidden strategies
```

> - [Puzzle SDK for zkGames](https://docs.puzzle.online/tutorials/build_your_first_zkgame/) purpose built SDK for building multiparty hidden information games on Aleo by the Puzzle Wallet team 

Reviewer: ```suggestion
- [Puzzle SDK for zkGames](https://docs.puzzle.online/tutorials/build_your_first_zkgame/) - A SDK for building multiparty hidden information games on Aleo
```
